testing/test_collection.py

```diff
@@ -1186,3 +1186,56 @@ def test_collect_pkg_init_and_file_in_args(testdir):
             "*2 passed in*",
         ]
     )
+
+
+@pytest.mark.skipif(
+    not hasattr(py.path.local, "mksymlinkto"),
+    reason="symlink not available on this platform",
+)
+def test_collect_pkg_with_symlinks(testdir):
```

Comment:
Will parametrize this test, but waiting for feedback from Windows users, where the issue might be something else.

Reply:
Done.